app/(candidate)/dashboard/website/create/components/WebsiteCreateFlow.js

```diff
@@ -26,6 +27,27 @@ export default function WebsiteCreateFlow() {
   const [previewOpen, setPreviewOpen] = useState(false)
   const [step, setStep] = useState(1)
   const [saveLoading, setSaveLoading] = useState(false)
+  const [contactFieldsCleared, setContactFieldsCleared] = useState(false)
+  const [privacyModalOpen, setPrivacyModalOpen] = useState(false)
+
+  // Clear auto-filled contact information when website is first loaded
+  useEffect(() => {
+    if (website && !contactFieldsCleared && website.content?.contact) {
+      // Clear contact fields that may have been auto-filled by the backend
```

Comment:
non-blocking: 

Commenting is overly verbose and adds noise to the code instead of signal.

Both comments seem redundant to just reading what these few lines of code say, but ultimately we don't need _two_ comments saying what amounts to be the same thing.